css-overflow-4/Overview.bs

```diff
@@ -947,28 +952,33 @@ Forcing a Break After a Set Number of Lines: the 'max-lines' property</h3>
 		Name: max-lines
 		Value: ''none'' | <<integer [1,∞]>>
 		Initial: ''none''
-		Applies to: <a>block containers</a> which are also [=fragmentation containers=] that capture [=region breaks=]
+		Applies to: <a>block containers</a> which are also either [=fragmentation containers=] that capture [=region breaks=] or [=line-clamp containers=]
 		Inherited: no
 		Percentages: N/A
 		Computed value: the keyword ''max-lines/none'' or an integer
 		Animation type: by computed value type
 	</pre>
 
-	This property only has an effect on boxes that are [=fragmentation containers=] that capture [=region breaks=].
+	If the value of 'max-lines' is not <dfn for=max-lines dfn-type=value>none</dfn>, then, where
+	<var>N</var> is the computed value of 'max-lines':
+
+	- If the box is a [=fragmentation container=] that captures [=region breaks=],
+	    a <a>region break</a> or
+	    is forced after its <var>N</var>th
+	    descendant <a>in-flow</a> <a>line box</a>.
+	    If fewer than <var>N</var> line boxes exist,
+	    then 'max-lines' introduces no <a>region break</a>.
+	- If the box is a [=line-clamp container=], its [=clamp point=] is set after its <var>N</var>th
+	    descendant <a>in-flow</a> <a>line box</a>.
+	    If fewer than <var>N</var> line boxes exist, then the clamp point is set to the end of the
```

Comment:
I added a clamp point at the end because, for `line-clamp: auto`, that is enough to explain the lack of visible clamping when the max-height is larger than the box's auto size. The last clamp point that fits is the clamp point at the end, which is visually equivalent to no clamping.

I don't feel strongly about this, though, so we could change this to not have a clamp point at the end, as long as we also say that the last line doesn't have a following clamp point (maybe unless it's followed by non-inline content, depending on whether we allow clamping between lineless blocks).